bin/ci

```diff
@@ -56,37 +83,79 @@ function update_sources() {
             echo "${site}")
 
         # shellcheck disable=SC2016
-        sed 's|${SITE}|'"${site}"'|g; s|${IMAGE}|'"${image}"'|g' \
-            < "${template}" >> ".circleci/src/workflows/@${site}.yml"
+        sed_cmd="s|\${SITE}|$site|g;s|\${IMAGE}|$image|g;"
+        if [ "$1" = true ] ; then
+            sed_cmd="$sed_cmd/CHANGELOG/,/lint-changelog/{//!d;};"
```

Comment:
Can you add a comment in english to describe what this does?

Reply:
Or maybe it is clearer to add a variable on line 10 to define ignore conditions?
So you can use the same pattern as for the site and image : search the variable and replace it by ignore conditions?

Reply:
The way I implemeted it is not robust at all.
So I wrapped check-changelog job between #!CHANGELOG_OPTION in template file
And if ignore-changelog is true, I delete all delete wrapped into these comments